created even_or_odd.md file Documentation in even_or_odd folder > src…

mimiprosper opened the pull request:
Closes #64 
I created an even_or_odd.md file in the even_or_odd folder > src folder.
In the even_or_odd.md file i wrote the documentation for the script, even_or_odd.cairo

danielcdz commented on cairo/scripts/even_or_odd/src/even_or_odd.md:
This looks great overall, but there are areas where we could enhance the explanations by adding references to the Cairo book. For instance:  
- When explaining variable assignments, reference relevant sections discussing variable declaration and scope.  
- Provide details about the different types in Cairo and their specific use cases, linking to the appropriate sections in the book.  
- Explain why the `return` reserved word can sometimes be omitted.  
- Additionally, look for other opportunities to add references or clarify concepts that might benefit from further context or examples from the Cairo book.

mimiprosper commented:
I have updated the even_or_odd documentation  to reflect the changes you asked for.

- I made references to relevant sections
- I added more context to the documentation
-  I made more in-depth explanation to some concepts

danielcdz reviewed (approved):
LGTM!
